src/santander_check_number_validator.js

```diff
@@ -0,0 +1,35 @@
+(function(window) {
+  var Moip = window.Moip || {};
+  window.Moip = Moip;
+
+  function SantanderCheckNumberCalculator() {
+    if ( !( this instanceof SantanderCheckNumberCalculator ) ) {
+      return new SantanderCheckNumberCalculator();
+    }
+  }
+
+  SantanderCheckNumberCalculator.prototype = {
+
+    calculateAccount: function(agencyNumber,accountNumber) {
+      var numbers = (agencyNumber + "00" + accountNumber).split("");
```

Comment:
You can remove this `"00"` from here if you also remove the `0, 0` from the array at line 27